tests/integration/jit/tracing/test_span_events_are_created_upon_caught_exception.php

```diff
@@ -33,16 +33,12 @@
 zend_extension=opcache.so
 */
 
-/*EXPECT_ERROR_EVENTS
-null
-*/
-
```

Comment:
Why was this removed? The 'baseline' for this test has this: https://github.com/newrelic/newrelic-php-agent/blob/acc7942cb672999fd87537be9aae508915bcc916/tests/integration/span_events/test_span_events_are_created_upon_caught_exception.php#L26-L28

Reply:
This behavior differs between PHP 8.0/8.1 and PHP 8.2 because PHP used to accept less strongly typed error handlers without explicit parameters. Starting with PHP 8.2, error handlers need to have explicit parameters.
As such, exceptions handled by old style error handlers will be correctly propagated for PHP 8.0/8.1,
but PHP 8.2 and above requires strongly typed error handlers otherwise the object it returns isn't explicitly recognizable as a Throwable.

The tests are split to accommodate the difference.

Please see: [d139ac6](https://github.com/newrelic/newrelic-php-agent/pull/715/commits/d139ac65b1e4d608579bae7120f5fd19d9e56392) which does the following:
* Split tests since PHP used to accept less strongly typed error handlers without explicit parameters.
Starting with PHP 8.2, error handlers need to have explicit parameters.
As such, exceptions handled by old style error handlers will be correctly propagated for PHP 8.0/8.1,
but PHP 8.2 and above requires strongly typed error handlers otherwise the object it returns isn't explicitly recognizable as a Throwable.
To cover both cases, there is now one test for PHP 8.0/8.1 with old style error handler and one test for PHP 8.0+ with new style handler.
* Updated description to explicitly determine if error events are created since previous description only noted if span events were created.
* Updated names to more accurately reflect test behavior.